docs/standards/FAQ/Channel-Nonce.md

```diff
@@ -0,0 +1,60 @@
+---
+title: What are multi-channel nonces
+sidebar_position: 2
+---
+
+
+This concept was taken from [Out of order execution](https://github.com/amxx/permit#out-of-order-execution).
+
+Using nonces prevent old signed transactions from being replayed again (replay attacks). A nonce is an arbitrary number that can be used just once in a transaction.
+
+#### Problem of Sequential Nonces
+
+With native transactions, nonces are strictly sequential. This means that messages with sequential nonces must be executed in order. For instance, in order for message number 4 to be executed, it must wait for message number 3 to complete.
+
+However, **sequential nonces come with the following limitation**:
+
+Some users may want to sign multiple message, allowing the transfer of different assets to different recipients. In that case, the recipient want to be able to use / transfer their assets whenever they want, and will certainly not want to wait on anyone before signing another transaction.
+
+ This is where **out-of-order execution** comes in.
+
+#### Introducing multi-channel nonces
+
+Out-of-order execution is achieved by using multiple independent channels. Each channel's nonce behaves as expected, but different channels are independent. This means that messages 2, 3, and 4 of `channel 0` must be executed sequentially, but message 3 of channel 1 is independent, and only depends on message 2 of `channel 1`.
+
+The benefit is that the signer key can determine for which channel to sign the nonces. Relay services will have to understand the channel the signer choose and execute the transactions of each channel in the right order, to prevent failing transactions.
+
+#### Nonces in the KeyManager
```

Comment:
Change this to header 2 or 3:

```suggestion
### Nonces in the KeyManager
```

so we get the titles in the sidebar.

Before:
![image](https://user-images.githubusercontent.com/477945/141319971-a546250b-f3dd-41ff-ad9f-b61c1a1e1899.png)


After:
![image](https://user-images.githubusercontent.com/477945/141319933-b7c41d90-5d33-4273-afad-743bdcd9158f.png)